packages/react-components/.storybook/main.js

```diff
@@ -1,4 +1,4 @@
-const isTestsRunner = process.env.NODE_ENV === 'PERCY' || process.env.NODE_ENV === 'cypress' // isPercy package doesn't work here
+const isTestsRunner = process.env.NODE_ENV === 'PERCY' || !!window.Cypress // is-tests-runner package doesn't work here
```

Comment:
@pshoniuk I'm getting a null reference on `window` when I run storybook on its own `yarn dev`

Reply:
Should be fixed